.expeditor/end_to_end.pipeline.yml

```diff
@@ -278,6 +278,17 @@ steps:
             - BUILD_PKG_TARGET=x86_64-windows
             - BUILDKITE_AGENT_ACCESS_TOKEN
 
+  - label: "[:linux: test_pkg_download]"
```

Comment:
Eyeballing the expeditor test timings, a great deal of time is setting up the environment compared to the actual test execution time. As we add more of these tests this might not scale super well; we might want to just add a batch script to run all / most of the tests in the end_to_end directory in a single test env.

Reply:
what I have noticed is the bulk of time can be spent in `docker pull` but sometimes the image is already there. I don't understand why sometimes it is there and sometimes not but ensuring the docker images are on the test VMs could significantly improve performance.

Reply:
Just to be clear, this whole "parallelization via Buildkite" approach was always intended at a near-term expedient. Eventually, we need to set up some kind of actual test runner code infrastructure to manage these tests.